client/src/main/java/io/pravega/client/tables/TableWriterConfig.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.client.tables;
+
+import java.io.Serializable;
+import lombok.Builder;
+import lombok.Data;
+
+/**
+ * Configuration for the TableWriter.
+ */
+@Data
+@Builder
+public class TableWriterConfig implements Serializable {
```

Comment:
This is currently just a copy of `EventWriterConfig`. if this class is always a superset of `EventWriterConfig`, then perhaps it should extend it.

Reply:
These will be disjoint and will contain Table-specific config. It's just a stub. Eventually it will evolve into its own thing, and I find no need to join it with EventWriterConfig at this point based on that.

Reply:
To be precise, when you say that "Eventually it will evolve into its own thing", are you saying that it won't be a strict superset of `EventWriterConfig`? If it will, then we better extend it, and otherwise leaving as is sounds good.

Reply:
I do not know how this will change in the future. Until it does, I do not want to over engineer it.